tachyon/zk/plonk/halo2/proof_reader.h

```diff
@@ -40,7 +40,7 @@ class ProofReader {
  public:
   using F = typename PCS::Field;
   using C = typename PCS::Commitment;
```

Comment:
Some use `C` and others use `Commitment`. What are the criteria for using each?

```suggestion
  using Commitment = typename PCS::Commitment;
```

Reply:
Not determined, but it would be nice to handle this issue with https://github.com/kroma-network/tachyon/issues/174!